src/applications/vaos/components/BackLink.jsx

```diff
@@ -14,53 +15,30 @@ export default function BackLink({ appointment }) {
   } = appointment.vaos;
 
   const location = useLocation();
+  const history = useHistory();
   const dispatch = useDispatch();
 
-  const handleClickGATracker = () => {
-    let status;
-    if (isPendingAppointment) {
-      status = 'pending';
-    } else if (isUpcomingAppointment) {
-      status = 'upcoming';
-    } else if (isPastAppointment) {
-      status = 'past';
-    }
+  let status;
+  let link = '/';
+  let linkText = 'Back to appointments';
+  if (isPendingAppointment) {
+    status = 'pending';
+    link = '/pending';
+    linkText = 'Back to request for appointments';
+  } else if (isUpcomingAppointment) {
+    status = 'upcoming';
```

Comment:
nit: Hmm it's a bit strange that we set just status here and rely on the defaults on lines 22-23. Let's put all three in the same spot, either all at the beginning or all in this block.

Reply:
Good point. I'll change the assignment to be in the same block.